tsl/test/sql/chunk_utils_internal.sql

```diff
@@ -440,6 +440,57 @@ CALL refresh_continuous_aggregate('ht_try_weekly', '2019-12-29', '2020-01-10', f
 SELECT * FROM ht_try_weekly;
 DROP MATERIALIZED VIEW ht_try_weekly;
 
+\c :TEST_DBNAME :ROLE_SUPERUSER
```

Comment:
why do need this mapping? It is already setup for one of the ROLES.
what needs superuser access for the test case?

Reply:
That was a leftover from the previous iteration. I removed it.